sickrage/ws/MedusaWebSocketHandler.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+# Author: p0psicles
+# URL: http://code.google.com/p/sickbeard/
+#
+# This file is part of SickRage.
+#
+# SickRage is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# SickRage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with SickRage. If not, see <http://www.gnu.org/licenses/>.
+
+
+import json
+import time
+from tornado.websocket import WebSocketHandler
+
+clients = []
+
+
+def push_to_web_socket(msg):
+    for client in clients:
+        client.write_message(msg)
+
+
+class WebSocketUIHandler(WebSocketHandler):
+    def check_origin(self, origin):
+        return True
```

Comment:
This will be fine since we don't accept anything server side. This is really only a XSS issue and since we only broadcast to the client there's nothing that XSS could cause.